src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -160,13 +162,14 @@ export default {
 			enableRecommendedReplacementExp: false,
 			featuredLoan: {},
 			showFiveDollarsBanner: false,
+			enableThreeLoansRecommended: false,
 			HandOrangeIcon,
 		};
 	},
 	apollo: {
 		preFetch(config, client) {
 			return client.query({
-				query: experimentAssignmentQuery, variables: { id: RECOMMENDED_REPLACEMENT_EXP_KEY }
+				query: experimentAssignmentQuery, variables: { id: THREE_LOANS_RECOMMENDED_ROW_EXP_KEY }
```

Comment:
Don't we still need to fetch `RECOMMENDED_REPLACEMENT_EXP_KEY`? I see it being used in the `.then` of this query. Although I guess we doing a global prefetch of assignments, and none of these assignment calls are needed in components? Please verify locally.

Reply:
We don't need to fetch `RECOMMENDED_REPLACEMENT_EXP_KEY` more.
For that reason we have other ticket to remove this setup as mentioned in the description above.